cohort-engine/src/main/java/com/ibm/cohort/engine/BooleanEvaluationCallback.java

```diff
@@ -0,0 +1,49 @@
+/*
+ *
+ *  * (C) Copyright IBM Corp. 2021
+ *  *
+ *  * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package com.ibm.cohort.engine;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.slf4j.Logger;
+
+import org.slf4j.LoggerFactory;
+
+public class BooleanEvaluationCallback implements EvaluationResultCallback {
+
+
+	public List<String> getPassingPatients(){
+		return passingPatients;
+	}
+
+	private List<String> passingPatients = new ArrayList<>();
+	private static final Logger logger = LoggerFactory.getLogger(BooleanEvaluationCallback.class.getName());
+
+	@Override
+	public void onContextBegin(String contextId) {
+		logger.info("Context: " + contextId);
+	}
+
+	@Override
+	public void onContextComplete(String contextId) {
+		logger.info("---");
```

Comment:
I think logging `contextId` on completion would be a good idea.
Since the server will be able to handle multiple concurrent evaluation requests, it would be nice to know which completion log line related to which `contextId`.

Reply:
done